src/StartScreen.java

```diff
@@ -0,0 +1,64 @@
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Scanner;
+
+public class StartScreen {
+    private HashMap<Integer, Car> startCars = new HashMap<>();
+
+    public Player getNewPlayer() {
+        System.out.println("Enter your name");
+        return new Player(inputLine(), 20000, showStartCars());
+    }
+
+    private Car showStartCars() {
+        CarService carS = new CarService();
+        System.out.println("Available cars:");
+        Iterator<Car> iterator = carS.getStartCars().iterator();
+        int i = 1;
+        while (iterator.hasNext()) {
+            startCars.putIfAbsent(i, iterator.next());
+            System.out.println(i + ". " + startCars.get(i).getModel() + " - " + startCars.get(i).getCarCost() + "$");
+            i++;
+        }
```

Comment:
in a while methods like this should be inside carService and StartScreen should use this methods with adding output/input activites